### PR TITLE
Fix almost all problems on PHPStan level 1

### PR DESCRIPTION
**Pull request by szepeviktor**
@iamgergo Please fix [the remaining 5](https://github.com/thepinecode/bazar/pull/9/checks?check_run_id=1087817075#step:6:15). Larastan gets Model properties from migrations.
So I think `Bazar\Models\Cart::$currency` is really missing.

Would you be so kind to explain the false positives to me? I'm eager to learn. Thanks.

### CODE REVIEW

**Comment by szepeviktor** [comment]
:) I think I may receive the answer _We don't use that method there_ :)

File: src/Models/User.php [hunk 1]
```diff
@@ -16,10 +16,6 @@
 use Illuminate\Support\Facades\Hash;
 use Illuminate\Support\Str;
 
-/**
- * @property string $name
- * @property string $email
- */
```
Comment (szepeviktor) [inline src/Models/User.php]: @iamgergo This cannot be shimmed :(

**Comment by szepeviktor** [comment]
I leave it up to You to manage this project and its testing.

**Comment by iamgergo** [comment]
https://github.com/thepinecode/bazar/commit/b73e862df08400bbbf1b9acb0df81f6d92065196

We are just trying to find a way how not to "pollute" the code with unnecessary comments.

Thanks!

**Comment by szepeviktor** [comment]
Okay, I understand that.

Up to Larastan/PHPStan 1.0 you **have** to use comments in many places. But I'm not able to describe how huge the benefit of PHPStan is! (compared to unit tests and human efforts)

**Comment by szepeviktor** [comment]
@iamgergo There is already a fork of the project ❗ https://github.com/szepeviktor/bazar/actions/runs/245996253

**Pull request closed without merging**